pkgs/by-name/gi/git-lfs-transfer/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule {
+  pname = "git-lfs-transfer";
+  version = "unstable-2024-10-07";
```

```suggestion
  version = "0.1.0-unstable-2024-10-07";
```

[Version should start with a digit](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning), last version was 0.1.0.